src/index.html

```diff
@@ -5,8 +5,242 @@
+          </p>
+
+          <a href="#buy" class="card__buy" data-qa="hover">Купить</a>
+        </article>
+        <article class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="iMac" class="card__image">
+          <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__feedback-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__feedbacks-count">Отзывов: 5</p>
+          </div>
+
+          <p class="card__price-container">
+            <span class="card__price-label">Цена:</span>
+            <span class="card__price">69 999 грн</span>
+          </p>
+
+          <a href="#buy" class="card__buy" data-qa="hover">Купить</a>
+        </article>
+        <article class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="iMac" class="card__image">
+          <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__feedback-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__feedbacks-count">Отзывов: 5</p>
+          </div>
+
+          <p class="card__price-container">
+            <span class="card__price-label">Цена:</span>
+            <span class="card__price">69 999 грн</span>
+          </p>
+
+          <a href="#buy" class="card__buy" data-qa="hover">Купить</a>
+        </article>
+        <article class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="iMac" class="card__image">
+          <h3 class="card__title">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</h3>
+          <p class="card__code">Код товара: 195434</p>
+
+          <div class="card__feedback-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__feedbacks-count">Отзывов: 5</p>
+          </div>
+
+          <p class="card__price-container">
+            <span class="card__price-label">Цена:</span>
+            <span class="card__price">69 999 грн</span>
+          </p>
+
+          <a href="#buy" class="card__buy" data-qa="card-hover">Купить</a>
```

Comment:
Delete data-qa="card-hover" in all places, you should add it only to the link inside the first card